test/AuthSamples.FunctionalTests/CustomPolicyProviderTests.cs

```diff
@@ -52,5 +119,26 @@ public CustomPolicyProviderTests(WebApplicationFactory<CustomPolicyProvider.Star
             Assert.Equal(HttpStatusCode.Redirect, response.StatusCode);
             Assert.Equal("http://localhost/account/signin?ReturnUrl=%2FHome%2FMinimumAge50", response.Headers.Location.ToString());
         }
+
+        internal static async Task<string> SignIn(HttpClient client, string userName, string dob)
```

Comment:
Its a little gross to have SignIn return the cookie that we then have to pass along to subsequent requests.  But if there's no way for it to just automatically flow, it is what it is...

Reply:
> Its a little gross to have SignIn return the cookie that we then have to pass along to subsequent requests

You can just call CreateClient() instead and that will give you a client that does cookies and redirects in the line below:
https://github.com/aspnet/AuthSamples/pull/41/files#diff-da74d3aa55a8bcfebb7ded6125d364e2L16